wire-ios-request-strategy/Sources/Payloads/Processing/ConversationEventPayloadProcessor.swift

```diff
@@ -408,7 +412,9 @@ final class ConversationEventPayloadProcessor {
         updateConversationTimestamps(for: conversation, serverTimestamp: serverTimestamp)
         updateConversationStatus(from: payload, for: conversation)
         updateMessageProtocol(from: payload, for: conversation)
-        updateMLSStatus(from: payload, for: conversation, context: context, source: source)
+        Task {
```

Comment:
Same here I would have expected `updateOrCreateGroupConversation()` to turn async